Sources/StreamChat/Models/Attachments/ChatMessageAudioAttachment.swift

```diff
@@ -47,6 +47,18 @@ public struct AudioAttachmentPayload: AttachmentPayload {
 
 extension AudioAttachmentPayload: Hashable {}
 
+// MARK: - Local Downloads
+
+extension AudioAttachmentPayload: AttachmentPayloadDownloading {
+    public var localStorageFileName: String {
+        title ?? file.defaultLocalStorageFileName(for: Self.type)
+    }
+    
+    public var remoteURL: URL {
+        audioURL
+    }
+}
```

Comment:
Do we support downloading audio files? If not, better to not conform to the protocol for now

Reply:
Not in the default UI, but no reason why one should not be able to do so in their custom UI.

Reply:
Yeah that is true 👍